gossipd/gossip.c

```diff
@@ -1369,7 +1371,11 @@ static void gossip_send_keepalive_update(struct routing_state *rstate,
 	status_trace("Sending keepalive channel_update for %s",
 		     type_to_string(tmpctx, struct short_channel_id, &scid));
 
-	handle_channel_update(rstate, update);
+	err = handle_channel_update(rstate, update);
```

Comment:
This should probably be fatal, it's an update we generate so if we don't accept it, that's on us.

Reply:
I'm generally trying to cut down on fatal errors, especially now we have status_broken().  But perhaps I'm premature: will change to fatal().

Reply:
Actually, it *is* fatal (when gossipd dies, main daemon dies):

```
	if (err)
		status_failed(STATUS_FAIL_INTERNAL_ERROR,
			      "rejected keepalive channel_update: %s",
			      tal_hex(trc, err));
```